cime_config/tests.py

```diff
@@ -133,7 +133,7 @@
             "SMS.T62_oQU120_ais20.MPAS_LISIO_TEST",
             "SMS.f09_g16_a.IGELM_MLI",
             "SMS_P12x2.ne4_oQU240.A_WCYCL1850.allactive-mach_mods",
-            "ERS_Ln9.ne4pg2_ne4pg2.F-MMF1.eam-mmf_crmout",
+            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMF1.eam-mmf_crmout",
```

Comment:
You cannot specify 96 procs here because we run developer on mappy, which does not have 96 procs.

Reply:
If you want to run on that many, you could move the test to integration.

Reply:
How many tasks can mappy run with? We could just set it to a lower value. We mainly just need to ensure that threading is disabled (until the thread fix gets merged).

Reply:
Actually, the more I think about this I should just cancel this PR, wait till #4333 gets merged and then make sure that the MMF tests don't specify the task count at all.

Reply:
I think the main point was to force the number of threads to be 1, not to specify the task count. Could temporarily add an `xmlchange NTHRDS=1` to the test mod but not a bad idea to just wait until threading is fixed. Alternatively I think you could add machine default PR layouts for F-MMF cases that specify 1 thread.